### PR TITLE
(vscode-ember-frost) Added new vscode extension

### PR DESCRIPTION
**Pull request by gep13**

### CODE REVIEW

File: manual/vscode-extensions/vscode-ember-frost/vscode-ember-frost.nuspec [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>vscode-ember-frost</id>
+    <title>Ember Colorizer and Theme Extension</title>
+    <version>1.0.0</version>
+    <authors>Ciena Blue Planet</authors>
+    <owners>gep13 chocolatey</owners>
```
Comment (pascalberger) [inline manual/vscode-extensions/vscode-ember-frost/vscode-ember-frost.nuspec]: Owners should be a comma separated list as per [.nuspec Reference](https://docs.nuget.org/ndocs/schema/nuspec)

File: manual/vscode-extensions/vscode-ember-frost/vscode-ember-frost.nuspec [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    <projectUrl>https://marketplace.visualstudio.com/items?itemName=ciena-blueplanet.vsc-ember-frost</projectUrl>
+    <projectSourceUrl>https://github.com/ciena-blueplanet/vscode-ember-colorizer</projectSourceUrl>
+    <packageSourceUrl>https://github.com/chocolatey/chocolatey-coreteampackages/tree/master/manual/vscode-extensions/vscode-ember-frost</packageSourceUrl>
+    <docsUrl>https://github.com/ciena-blueplanet/vscode-ember-colorizer/blob/master/README.md</docsUrl>
+    <bugTrackerUrl>https://github.com/ciena-blueplanet/vscode-ember-colorizer/issues</bugTrackerUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <summary>Visual Studio Code Ember Colorizer and Theme Extension</summary>
+    <description>VsCode extension that colorizes/tokeninzes Ember .hbs, Controllers, and Routes files.</description>
+    <tags>Microsoft VisualStudioCode vscode Ember Extension foss</tags>
```
Comment (pascalberger) [inline manual/vscode-extensions/vscode-ember-frost/vscode-ember-frost.nuspec]: Tags in lowercase as per [§1.2.2](https://github.com/chocolatey/chocolatey-coreteampackages/wiki/Contributing-guidelines#122-obligatory-tags)

**Comment by gep13** [comment]
@pascalberger thank you for the review.  I have implemented all the suggestions.

File: manual/vscode-extensions/vscode-ember-frost/vscode-ember-frost.nuspec [hunk 3]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>vscode-ember-frost</id>
+    <title>Ember Colorizer and Theme Extension</title>
+    <version>1.0.0</version>
+    <authors>Ciena Blue Planet</authors>
+    <owners>gep13, chocolatey</owners>
```
Comment (pascalberger) [inline manual/vscode-extensions/vscode-ember-frost/vscode-ember-frost.nuspec]: Small one: chocolatey should be listed first by [§1.2.4](https://github.com/chocolatey/chocolatey-coreteampackages/wiki/Contributing-guidelines#124-add-chocolatey-among-owners)

Comment (gep13) [inline manual/vscode-extensions/vscode-ember-frost/vscode-ember-frost.nuspec]: Great catch, done!

**Pull request merged**